qa/distros/supported-all-distro/ubuntu_20.04.yaml

```diff
@@ -0,0 +1 @@
+../all/ubuntu_20.04.yaml
```

Comment:
https://github.com/ceph/ceph/pull/53517 removed 20.04 as a supported distro for the squid release, so it shouldn't be added back. if this piece is needed on reef, it shouldn't go through main first

Reply:
oops, this is a pr against reef :)